src/lib.rs

```diff
@@ -218,3 +218,48 @@ pub trait DCT6And7<T: common::DCTnum> : DCT6<T> + DCT7<T> {}
 
 /// A trait for algorithms that can compute both DST6 and DST7, all in one struct
 pub trait DST6And7<T: common::DCTnum> : DST6<T> + DST7<T> {}
+
+
+#[test]
+fn test_send_sync_impls() {
```

Comment:
I dunno if you consider this test block superfluous, but I figured I'd add it for posterity.